views/users/edit.erb

```diff
@@ -8,21 +8,35 @@
   <p>
     First name:
     <br />
-    <input type="text" name="fname" value="<%= @user[:fname] %>">
+    <input type="text" name="fname" value="<%= @user[:fname] %>" >
```

Comment:
To avoid repeating this whole form in your `users/new.erb` view, you can use a 'partial' where you put the form code and then call it from this page and from `users/new.erb` with `<%= erb :"users/_form" %>`. `_form` is just a naming convention for partials so we can easily identify partials in our views folders.